src/azure-cli/azure/cli/command_modules/container/_params.py

```diff
@@ -66,6 +66,7 @@ def load_arguments(self, _):
         c.argument('ports', type=int, nargs='+', default=[80], help='A list of ports to open. Space-separated list of ports')
         c.argument('protocol', arg_type=get_enum_type(ContainerNetworkProtocol), help='The network protocol to use')
         c.argument('dns_name_label', help='The dns name label for container group with public IP')
+        c.argument('auto_generated_domain_name_label_scope', arg_type=get_enum_type(AutoGeneratedDomainNameLabelScope), help='The autogenerate name label policy chosen by user')
```

Comment:
The name of this parameter seems too long for users to input. Could you give it a short alias ?